CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CanHazFunny
+{
+    public class Jester
+    {
+        private IJokeOutput outputService;
+        private IJokeService jokeService;
```

Comment:
Field should be prefixed with an underscore and PascalCase

```suggestion
        private IJokeOutput _OutputService;
        private IJokeService _JokeService;
```